js/kucoin.js

```diff
@@ -1974,77 +1908,40 @@ module.exports = class kucoin extends Exchange {
             throw new ExchangeError (this.id + ' type must be one of ' + keys.join (', '));
         }
         params = this.omit (params, 'type');
-        if ((type === 'contract') || (type === 'futures')) {
-            // futures api requires a futures apiKey
-            // only fetches one balance at a time
-            // by default it will only fetch the BTC balance of the futures account
-            // you can send 'currency' in params to fetch other currencies
-            // fetchBalance ({ 'type': 'futures', 'currency': 'USDT' })
-            const response = await this.futuresPrivateGetAccountOverview (params);
```

Comment:
This only returns one currency, so I don't think there is a fetchAccounts for futures